c2corg_api/search/search.py

```diff
@@ -44,10 +44,11 @@ def do_multi_search_for_types(search_types, search_term, limit, lang):
 
     for search_type in search_types:
         (_, get_documents_config) = search_type
-        search = create_search(get_documents_config.document_type).\
-            query(get_text_query_on_title(search_term, lang)).\
-            fields([]).\
-            extra(from_=0, size=limit)
+        search = (create_search(get_documents_config.document_type)
```

Comment:
It avoids the ugly `\` I guess.

Reply:
yes, the syntax of python allows free line breaks in a () block.
we used to have a coding convention to avoid the \ line continuation, so I instincly changed the block.
are there any style conventions in the project, except for standard linter ?

Reply:
No convention, and if we want one, we'll put it in the linter. So feel free.